src/components/Ambassadors/Company.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import './Ambassadors.scss';
+
+export const Company = ({ company }) => {
+    return (
```

Comment:
you could use destructuring here, const { name, address, website, email, phone, link, image } = company; and use it as address instead of company.address